config/main.py

```diff
@@ -929,7 +929,11 @@ def config(ctx):
         platform.add_command(mlnx.mlnx)
 
     # Load the global config file database_global.json once.
-    SonicDBConfig.load_sonic_global_db_config()
+    num_asic = multi_asic.get_num_asics()
+    if num_asic > 1:
+        SonicDBConfig.load_sonic_global_db_config()
```

Comment:
@Qi, we could call SonicDBConfig.initializeGlobalConfig() as well here ?

Reply:
I translated the names in https://github.com/Azure/sonic-swss-common/blob/9e91e0d891398b468b8087682ae91335791fac51/common/dbconnector.h#L57-L58

Reply:
Thanks Qi, I had noticed that .. wondering if you added that because of the "namespace" parameter which is not there in SonicDBConfig.initializeGlobalConfig(), if so we could add that.